treetime/distribution.py

```diff
@@ -49,6 +49,11 @@ def calc_fwhm(distribution, is_neg_log=True):
         # 0.69... is log(2), there is always one value for which this is true since
         # the minimum is subtracted
         tmp = np.where(log_prob < 0.693147)[0]
+        if len(tmp)==0:
+            raise Exception("Error in computing the FWHM for the distribution. This is "
```

Comment:
Maybe you could make this easier to understand for users? FWHM is probably full width half maximum - but for an end user this is still quite cryptic.

Reply:
Is it possible to know more specifically on which branch/node this error occurs? That could be useful for debugging the tree!